extensions/amp-3d-gltf/0.1/amp-3d-gltf.js

```diff
@@ -151,7 +151,7 @@ export class Amp3dGltf extends AMP.BaseElement {
     }
 
     const iframe = getIframe(this.win, this.element, '3d-gltf', this.context_);
-
+    iframe.setAttribute('title', this.element.title || 'GLTF 3D model');
```

Comment:
Nit (for this and the reset):

```suggestion
    iframe.title = this.element.title || 'GLTF 3D model';
```